src/utils/FourWay.js

```diff
@@ -369,12 +372,7 @@ class FourWay {
         info.settingsArray = (await this.read(eepromOffset, info.layoutSize)).params;
         info.settings = Convert.arrayToSettingsObject(info.settingsArray, info.layout);
 
-        /**
-         * If not AM32, then very likely BLHeli_32, even if not - we can't
-         * handle it.
-         */
-        const validNames = source.getValidNames();
-        if(!validNames.includes(info.settings.NAME)) {
+        if(!Object.values(am32Eeprom.BOOT_LOADER_PINS).includes(info.meta.input)) {
```

Comment:
Please leave the comment so we know what why we are doing this.

Reply:
readded